roles/ovirt-vm-infra/tasks/create_vms.yml

```diff
@@ -4,7 +4,9 @@
     pattern: "name={{ current_vm.name }}"
 
 - name: "Create VM {{current_vm.name }}"
-  ovirt_vms:
+  # TODO: Change when Ansible 2.4 is released:
+  # ovirt_vms:
+  ovirt_vms_24:
```

Comment:
Why not ovirt_vms_internal_24?

Reply:
Ah, no reason, forget to add. It's not an issue, because the module is named `ovirt_vms_24`. I've tested and it was executed successfully.

Reply:
Sure, just asked :-)